.github/workflows/dev-release.yml

```diff
@@ -0,0 +1,48 @@
+name: Release @dev tag to npm
+on:
+  pull_request:
+    types: [labeled]
+
+jobs:
+  release:
+    if: ${{ contains(github.event.pull_request.labels.*.name, 'release dev tag') }}
+    runs-on: ubuntu-22.04
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - name: Setup PNPM
+        uses: pnpm/action-setup@a3252b78c470c02df07e9d59298aecedc3ccdd6d # v3.0.0
+        with:
+          run_install: |
+            - args: [--frozen-lockfile]
+      - name: Check for changeset
+        run: pnpm exec changeset status --since origin/main
```

Comment:
is this doing anything except error?
https://github.com/changesets/changesets/blob/main/docs/command-line-options.md#status

If no changeset is there, it will kill the workflow. Should it? Maybe it should gracefully exit?

Reply:
Yes - it will fail. Idea here is that should prevent releasing dev tag if you don't have changeset inside your PR as releasing won't work anyway.

Reply:
Yea, but it will also send me an email that workflow failed. I think it should be my decision here - I don't want changeset (I edited readme for example), so this PR should just gracefully stop, not fail

Reply:
Yes - but it will be your decision to add `label` anyway 😄 . This workflow will only trigger if you add label so I think we need this check in place to avoid situation when you want to release snapshot without changesets 🤔